server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -4,6 +4,54 @@
  */
 
 export interface paths {
+  '/queue-admin/retry-dlq/{dlqName}': {
```

Comment:
I had to re-import the API swagger spec to get the data types for CreateEducationRequest etc.
But it also pulled in these changes to do with DLQ endpoints. These are not related to what we are doing in this PR, but it is not a problem. It just means we've not re-imported the spec for a while and there have been some unrelated changes happening in the API codebase recently.